backend/src/main/java/com/rota/api/StaffService.java

```diff
@@ -61,4 +61,19 @@ public Staff createStaff(Staff newStaff) {
         .filter(staff -> !staff.isInactive())
         .collect(Collectors.toList());
   }
+
+  /**
+   * Get a user object from database via email.
+   *
+   * @param email the user email
+   * @return the {@link Staff} member
+   */
+  public Staff findStaffByEmail(String email) {
+    return staffRepository.findAll().stream()
+        .filter(staff -> staff
+            .getEmail()
+            .equalsIgnoreCase(email))
+        .findFirst()
+        .orElse(null);
```

Comment:
Instead of having orElse(null), I would advise to have the method return Optional<Staff> and use `Optional.ofNullable`. This would immediately indicate to whoever wants to use this method that getting staff by email may fail.

Reply:
Yep this sounds good, I'll make the change!